page_analyzer/app.py

```diff
@@ -1,8 +1,76 @@
-from flask import Flask, render_template
+import os
+from typing import Any, Dict
+
+from dotenv import load_dotenv
+from flask import Flask, flash, redirect, render_template, request, url_for
+from page_analyzer.utils.db import connect_db
+from page_analyzer.utils.url import normalize, validate
+
+load_dotenv()
+
 
 app = Flask(__name__)
+app.secret_key = os.getenv("SECRET_KEY")
+
+
+def get_url_context(url_id: int) -> Dict[str, str]:
```

Comment:
[optional] you could also declare the return type as `dict[str, str | int]` and not cast `url_id` to string

Reply:
![Снимок экрана от 2024-01-30 22-05-33](https://github.com/AlexMomot-717/python-project-83/assets/76219029/cb9deccb-3a30-4c2c-b012-b1697fa10e6d)

Reply:
it's probably happening because `utils` is not a package
let's add `__init__.py` module and see what happens

Reply:
it is happening because the `context` variable is defined two times:
- the first declaration (where you assign values) clearly says that it is a dict[str, int | str]
- the second declaration is just an empty dict and mypy can't grasp it's type - that's why the final type is `dict[str, object]`

this can be fixed by adding type hint to the second declaration:

```py
context: dict[str, int | str] = {}
```

or by function restructuring